source/extensions/extensions_metadata.yaml

```diff
@@ -268,6 +268,11 @@ envoy.filters.http.squash:
   - envoy.filters.http
   security_posture: requires_trusted_downstream_and_upstream
   status: stable
+envoy.filters.http.sxg:
+  categories:
+  - envoy.filters.http
+  security_posture: unknown
```

Comment:
I think we actually want `robust_to_untrusted_downstream`, given we need to trust the upstream to sign its payload no?